### PR TITLE
[PPML] Add Azure Attestation part to Attestation Basic Docs

### PR DESCRIPTION
**Pull request by xiangyuT**
## Description

Add description of Azure Attestation and fix some typos.

### CODE REVIEW

File: docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md [hunk 1]
```diff
@@ -71,10 +71,20 @@ With this attestation service design, we can avoid adding malicious applications
...
 #### Attestation Service for Cloud Service Provider (CSP)
 
+Before we submit our applications to a cloud service, we need to verify the identity and security posture of the platform. Azure Attestation receives evidence from the platform, validates it with security standards, evaluates it against configurable policies, and produces an attestation token for claims-based applications.
+
+The involved actors in Azure Attestation workflow:
+* Relying party: The component which relies on Azure Attestation to verify enclave validity.
+* Client: The component which collects information from an enclave and sends requests to Azure Attestation.
+* Azure Attestation: The component which accepts enclave evidence from client, validates it and returns attestation token to the client
+
+![Azure Attestation Workflow](../images/azure_attestation_workflow.png)
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md:81]: Can we directly refer azure's image instead of import it to our repo?

Comment (xiangyuT) [reply docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md]: Refined.

File: docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md [hunk 2]
```diff
@@ -71,10 +71,20 @@ With this attestation service design, we can avoid adding malicious applications
 
 #### Attestation Service for Cloud Service Provider (CSP)
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md:72]: from Cloud ...

File: docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md [hunk 3]
```diff
@@ -71,10 +71,20 @@ With this attestation service design, we can avoid adding malicious applications
...
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md:73]: Add a sentence:
Azure provides an Attestation Service for applications running in TEE VM or containers provided by Azure.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**